fastlane/Pluginfile

```diff
@@ -7,8 +7,8 @@ group :screenshots, optional: true do
 end
 
 # This comment avoids typing to switch to a development version for testing.
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: ''
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'fix-ios-linter'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.0'
```

Comment:
🚨  We need to point back to `trunk` or to the next release once https://github.com/wordpress-mobile/release-toolkit/pull/353 lands before merging this PR